### PR TITLE
(DOCSP-29251, DOCSP-29252, DOCSP-29253): Insert Usage Example pages

### PR DESCRIPTION
**Pull request by cbullinger**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-29251
Staging - 
- https://docs-mongodbcom-staging.corp.mongodb.com/kotlin/docsworker-xlarge/docsp-29251-insert-usage/usage-examples/insertOne/
- https://docs-mongodbcom-staging.corp.mongodb.com/kotlin/docsworker-xlarge/docsp-29251-insert-usage/usage-examples/insertMany/

## Updated Pages
- Usage Examples > Insert Operations > [Insert a Document page](https://www.mongodb.com/docs/drivers/java/sync/current/usage-examples/insertOne/)
- Usage Examples > Insert Operations > [Insert Multiple Documents page](https://www.mongodb.com/docs/drivers/java/sync/current/usage-examples/insertMany/)

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?

### CODE REVIEW

File: examples/src/main/kotlin/usageExamples/insert/InsertOne.kt [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+// }
+// :snippet-start: insert-usage-example
+import com.mongodb.MongoException
+import com.mongodb.kotlin.client.coroutine.MongoClient
+import io.github.cdimascio.dotenv.dotenv
+import kotlinx.coroutines.runBlocking
+import org.bson.codecs.pojo.annotations.BsonId
+import org.bson.types.ObjectId
+
+data class Movie(@BsonId val id: ObjectId, val title: String, val genres: List<String>)
```
Comment (mongodben) [inline examples/src/main/kotlin/usageExamples/insert/InsertOne.kt:17]: why not use the same data class as insertMany() example here?

**Review by mongodben — APPROVED** [review]
lgtm w 1 small comment about a potential change

File: examples/src/main/kotlin/usageExamples/insert/InsertOne.kt (hunk 1 above)
Comment (rozza) [reply examples/src/main/kotlin/usageExamples/insert/InsertOne.kt:17]: You could do. It probably was simplified in insertMany example due to the verboseness of Java.

**Review by rozza — APPROVED** [review]
LGTM - feel free to use the same data class in insertMany should you wish.

**Pull request merged**